src/chat/index.tsx

```diff
@@ -456,6 +457,9 @@ export default (props: Chat) => {
         onSelectDefaultOption={
           props.footerOnSelectDefaultOption && props.footerOnSelectDefaultOption
         }
+        footerOnPressLaunchIcon={
+          props.footerOnPressLaunchIcon && props.footerOnPressLaunchIcon
+        }
```

Comment:
In the footer component, let this be named `onPressLaunchIcon` then in the `chat/index.tsx` in should be `footerOnPressLaunchIcon`

I know the codebase has a lot to improve on, like here we don't need to call `props.footerOnPressLaunchIcon && props.footerOnPressLaunchIcon`, we can just call it once, but let's leave it for consistency sake, refactoring would come in the future